src/pages/output/OutputProgramMicrobitView.svelte

```diff
@@ -99,6 +100,41 @@
       return s;
     });
   };
+
+  // LZMA isn't a proper module.
+  // When bundled it assigns to window. At dev time it works via the above import.
+  const LZMA = (window as any).LZMA ?? lzma.LZMA;
+
+  const handleExport = () => {
+    const compressed = LZMA.compress(
+      JSON.stringify({
+        meta: {
+          // PXT version specified and may need updating
```

Comment:
All the info needed is in the MakeCode about dialog (or a click or two away in GH) so we may as well get it right now.

Reply:
👍  Have updated them here: https://github.com/microbit-foundation/ml-trainer/pull/262/commits/3380fd2e8443b9b2bad5775952dec5d16681295f